test/src/system/list_formatting_test.coffee

```diff
@@ -68,3 +68,28 @@ testGroup "List formatting", template: "editor_empty", ->
                 assert.blockAttributes([2, 3], [])
                 assert.blockAttributes([3, 5], ["bulletList", "bullet"])
                 expectDocument("a\n\nc\n")
+
+  test "deleting from linebreak into a list should not remove the list element", (expectDocument) ->
+    typeCharacters "\n\n", ->
+      clickToolbarButton attribute: "bullet", ->
+        typeCharacters "a", ->
+          getSelectionManager().setLocationRange(index: 0, offset: 1)
+          getComposition().deleteInDirection("forward")
+          getSelectionManager().setLocationRange(index: 0, offset: 0)
+          getComposition().deleteInDirection("forward")
+          getEditorController().render()
+
+          assert.blockAttributes([0, 1], ["bulletList", "bullet"])
+          expectDocument("a\n")
```

Comment:
isn't that expectDocument("a")?

Reply:
maybe it is alway the default character?

Reply:
Yeah the \n gets added by default